proto/uber/cadence/api/v1/service_workflow.proto

```diff
@@ -90,6 +93,11 @@ service WorkflowAPI {
   rpc RefreshWorkflowTasks(RefreshWorkflowTasksRequest) returns(RefreshWorkflowTasksResponse);
 }
 
+message RestartWorkflowExecutionRequest {
+  string domain = 1;
+  WorkflowExecution workflow_execution = 2;
+  string identity = 3;
```

Comment:
What is identity?

Reply:
identity is the guy making the call. it gets passed in in all requests pretty much

Reply:
ah, sorry, I was expecting AuthN information to be passed as a header. But, as you say, this seems to be a field throughout the rest of the APIs. ok, fair.